src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -215,18 +216,53 @@ Copyright (c) .NET Foundation. All rights reserved.
                                    Condition="'%(_NETCoreNativeItems.FileName)%(_NETCoreNativeItems.Extension)' == '$(_DotNetHostPolicyLibraryName)' or
                                               '%(_NETCoreNativeItems.FileName)%(_NETCoreNativeItems.Extension)' == '$(_DotNetHostFxrLibraryName)'" />
 
-      <NativeNETCoreCopyLocalItems Include="%(_NETCoreNativeItems.ResolvedPath)"
-                                   Condition="'%(_NETCoreNativeItems.FileName)%(_NETCoreNativeItems.Extension)' == '$(_DotNetHostExecutableName)'">
```

Comment:
We definitely need to ensure that this change lights-up in the SDK only for 2.0+ apps.